bin/worker.js

```diff
@@ -7,8 +7,14 @@
 require('../lib/newrelic')();
 
 const logger = require('../lib/logging')('bin.worker');
-const server = require('../lib/server/worker').create();
 
-server.start(function() {
-  logger.info('listening', server.info.uri);
-});
+async function start(){
+  const server = await require('../lib/server/worker').create();
+
+  server.start(function() {
```

Comment:
I don't think there is a callback anymore on `.start`, You need to do `await server.start()`
and surround it with the a try and catch to throw an error if it fails to `.start`